addons/website_forum/static/src/js/website_forum.editor.js

```diff
@@ -8,9 +8,9 @@ var Dialog = require('web.Dialog');
 var _t = core._t;
 
 var ForumCreateDialog = Dialog.extend({
-    xmlDependencies: Dialog.prototype.xmlDependencies.concat(
-        ['/website_forum/static/src/xml/website_forum_templates.xml']
-    ),
+    xmlDependencies: Dialog.prototype.xmlDependencies.concat([
+        '/website_forum/static/src/xml/website_forum_templates.xml',
+    ]),
```

Comment:
Useless diff

Reply:
This is a result of the FW, I removed the diff.

Reply:
The forward-port set this:
![image](https://user-images.githubusercontent.com/10338094/157433489-fbd80054-e589-4ad9-afa0-8437f2b7e6f8.png)

And you changed into this:
![image](https://user-images.githubusercontent.com/10338094/157433581-73cddb04-6049-4a85-9980-1119a01d51e8.png)

So that was not the result of the FW but a result from your own making :wink: I personally would not mind "useless" changes that improve code readability / convention / etc but when it does not change anything like that in this case, I guess it's better to care for the github history :+1: